packages/marble-api/openapis/marblecore-api.yaml

```diff
@@ -143,6 +143,15 @@ paths:
             items:
               type: string
               format: uuid
+        - name: inbox_id[]
```

Comment:
I'm wondering if I don't want to call it `case_inbox_id`, to clarify that it's not a field from the decision itself 🤔

Reply:
As you wish, I will wait for your backend PR before merging this, so I can adapt changes + manually test e2e the feature